[Docs] Indicate that CCR does not auto-follow existing indices, only newly created ones (#89498)
elastic/elasticsearch: docs/reference/ccr/apis/auto-follow/put-auto-follow-pattern.asciidoc

<<ccr-auto-follow,auto-follow patterns>> against the remote cluster
specified in the request body. Newly created indices on the remote cluster
matching any of the specified patterns will be automatically configured as follower

> indices. Indices on the remote cluster that have been created before the auto-follow
>  pattern is created won't be auto-followed even if they match the pattern.

Reviewer: Small fix:

```suggestion
indices. Indices on the remote cluster that were created before the auto-follow
pattern is created won't be auto-followed even if they match the pattern.
```

or perhaps this?

```suggestion
indices. Indices on the remote cluster that exist when the auto-follow
pattern is created won't be auto-followed even if they match the pattern.
```

Author: I borrowed the first suggestion, thanks David!
